packages/components/NativeRadioButton/macos/RadioButton.swift

```diff
@@ -0,0 +1,21 @@
+class RadioButton: NSButton {
+
+	@objc public var onPress:RCTBubblingEventBlock?
+	
+	@objc public func callOnPress() {
+		self.window?.makeFirstResponder(self)
+		if (onPress != nil) {
+			onPress!(nil)
+		}
+	}
+	
+	/// Set button state to 'on' when it becomes focused
+	override func becomeFirstResponder() -> Bool {
+		/// fire onChange event
```

Comment:
nit, // not /// since it's inline comments rather than documentation style comments